NSS-Day-Cohort-42/rare-media-moguls
# Rp header image

#### Changes Made
1. added file input to post form so that users can upload a header image when creating a post or editing
1.added header image to post detail with styling to size image
1. fixed bug with post tags when switching from editing a post to creating a new post


#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout rp-headerImage
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py server
```
4. Test app functionality.
> Create a new Post with an image
>edit a post and add an image
>navigate from editing a post to creating a new post and make sure everything still works with post tags

5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
